lib/configure.js

```diff
@@ -280,6 +280,43 @@ function configure (gyp, argv, callback) {
       }
     }
 
+    // For z/OS we need to set up the path to zoslib include directory,
+    // which contains headers included in v8config.h.
+    var zoslibIncDir
+    if (process.platform === 'os390') {
+      logprefix = "find zoslib's zos-base.h:"
+      let msg
+      var zoslibIncPath = process.env.ZOSLIB_INCLUDES
+      if (zoslibIncPath) {
+        zoslibIncPath = findAccessibleSync(logprefix, zoslibIncPath, ['zos-base.h'])
+        if (zoslibIncPath === undefined) {
+          msg = msgFormat('Could not find zos-base.h file in the directory set ' +
+                          'in ZOSLIB_INCLUDES environment variable: %s; set it ' +
+                          'to the correct path, or unset it to search %s', process.env.ZOSLIB_INCLUDES, nodeRootDir)
+        }
+      } else {
+        candidates = [
+          'include/node/zoslib/zos-base.h',
+          'include/zoslib/zos-base.h',
+          'zoslib/include/zos-base.h',
+          'install/include/node/zoslib/zos-base.h'
+        ]
+        zoslibIncPath = findAccessibleSync(logprefix, nodeRootDir, candidates)
+        if (zoslibIncPath === undefined) {
+          msg = msgFormat('Could not find any of %s in directory %s; set ' +
+                          'environmant variable ZOSLIB_INCLUDES to the path ' +
+                          'that contains zos-base.h', candidates.toString(), nodeRootDir)
+        }
+      }
+      if (zoslibIncPath !== undefined) {
+        zoslibIncDir = path.dirname(zoslibIncPath)
+        log.verbose(logprefix, "Found zoslib's zos-base.h in: %s", zoslibIncDir)
+      } else {
+        log.error(logprefix, msg)
```

Comment:
Can we make this apply to Node v16 and above since we don't ship zoslib in earlier versions?

Reply:
No because node-gyp should work on all supported versions of Node.js.